docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -146,28 +146,74 @@ _Triggers the **[DataChanged](#datachanged)** event when setting data successful
 
 | Name     | Type       | Description                       |
 | :------- | :--------- | :-------------------------------- |
-| `keys`   | bytes32[ ] | The keys for which to set value.  |
-| `values` | bytes[ ]   | The array of bytes values to set. |
+| `keys`   | bytes32[ ] | The keys for which to set data.   |
+| `values` | bytes[ ]   | The array of data to set.         |
 
 ### getData
 
 ```solidity
 function getData(bytes32[] memory keys) public view returns (bytes[] memory values)
 ```
 
-Retrieve an array of values for multiple given keys.
+Retrieve an array of data for multiple given keys.
 
 #### Parameters:
 
 | Name   | Type       | Description                       |
 | :----- | :--------- | :-------------------------------- |
-| `keys` | bytes32[ ] | The keys to retrieve values from. |
+| `keys` | bytes32[ ] | The keys to retrieve data from.   |
 
 #### Return Values:
 
 | Name     | Type     | Description                                    |
 | :------- | :------- | :--------------------------------------------- |
-| `values` | bytes[ ] | An array of the values for the requested keys. |
+| `values` | bytes[ ] | An array of the data for the requested keys. |
+
+
+### setData
+
+```solidity
+function setData(
+    bytes32 keys,
+    bytes memory value
+) public
+```
+
+Set data as **bytes** in the vault storage for a single key.
+
+_Triggers the **[DataChanged](#datachanged)** event when setting data successfully._
+
+:::note
+**The `setData(...)` function can only be called by the current owner of the contract and the LSP1UniversalReceiverDelegateVault contract.**
```

Comment:
maybe we can say this owner can be a Key Manager which then allows multiple "accounts / addresses" to still interact with the profile? or is it off context?

Reply:
I think it's already mentioned in the standards section

Reply:
ok 👌